src/types.ts

```diff
@@ -8,3 +8,51 @@ export interface MoviePreference {
   user_id: number;
   external_movie_id: string;
 }
+
+interface Genre {
+  id: number;
+  name: string;
+}
+
+interface ProductionCompany {
+  id: number;
+  logo_path: string;
+  name: string;
+  origin_company: string;
+}
+
+interface ProductionCountry {
+  iso_3166_1: string;
+  name: string;
+}
+
+interface SpokenLanguage {
+  iso_639_1: string;
+  name: string;
+}
+
+export interface ExternalMovie {
```

Comment:
Do we need all of these? Seems more brittle than adding them as we need them

Reply:
That's true - I can just keep the parts we care about